tests/slack_sdk_async/web/test_web_client_coverage.py

```diff
@@ -38,13 +38,18 @@ def setUp(self):
         self.api_methods_to_call = []
         for api_method in self.all_api_methods:
             if api_method.startswith("apps.permissions.") or api_method in [
+                "apps.connections.open",  # app-level token
+                "auth.teams.list",  # app-level token
                 "oauth.access",
                 "oauth.v2.access",
                 "oauth.token",
                 "users.setActive",
                 "admin.conversations.whitelist.add",  # deprecated
                 "admin.conversations.whitelist.listGroupsLinkedToChannel",  # deprecated
                 "admin.conversations.whitelist.remove",  # deprecated
+                "admin.conversations.getCustomRetention",
```

Comment:
What is the connection between these additions and the `admin.barriers.*` methods?